docs/CHANGELOG.md

```diff
@@ -1,47 +1,77 @@
 
-Enhancement:
-* Improved the performance of `aql.load_file` for files from AWS S3 to Google BigQuery up to 94%. [#429](https://github.com/astronomer/astro-sdk/issues/429), [#568](https://github.com/astronomer/astro-sdk/pull/568)
-* Improved the performance of `aql.load_file` for files from Google Cloud Storage to Google BigQuery up to 93%. [#429](https://github.com/astronomer/astro-sdk/issues/429), [#562](https://github.com/astronomer/astro-sdk/issues/562)
-* Improved the performance of `aql.load_file` for files from AWS S3/Google Cloud Storage to Snowflake up to 76%. [#430](https://github.com/astronomer/astro-sdk/issues/430), [#544](https://github.com/astronomer/astro-sdk/pull/544)
-* Improved the performance of `aql.load_file` for files from GCS to Postgres in K8s up to 93%. [#428](https://github.com/astronomer/astro-sdk/issues/428), [#531](https://github.com/astronomer/astro-sdk/pull/531)
-* Fix sphinx docs sidebar [#472](https://github.com/astronomer/astro-sdk/issues/472)
+### Enhancements
+* Improved the performance of `aql.load_file` for files for below:
+  * From AWS S3 to Google BigQuery up to 94%. [#429](https://github.com/astronomer/astro-sdk/issues/429), [#568](https://github.com/astronomer/astro-sdk/pull/568)
+  * From Google Cloud Storage to Google BigQuery up to 93%. [#429](https://github.com/astronomer/astro-sdk/issues/429), [#562](https://github.com/astronomer/astro-sdk/issues/562)
+  * From AWS S3/Google Cloud Storage to Snowflake up to 76%. [#430](https://github.com/astronomer/astro-sdk/issues/430), [#544](https://github.com/astronomer/astro-sdk/pull/544)
+  * From GCS to Postgres in K8s up to 93%. [#428](https://github.com/astronomer/astro-sdk/issues/428), [#531](https://github.com/astronomer/astro-sdk/pull/531)
 * Get configurations via Airflow Configuration manager. [#503](https://github.com/astronomer/astro-sdk/issues/503)
-* Add CI job to check for dead links [#526](https://github.com/astronomer/astro-sdk/pull/526)
+* Change catching `ValueError` and `AttributeError` to `DatabaseCustomError` [#595](https://github.com/astronomer/astro-sdk/pull/595)
+* Unpin pandas upperbound dependency [#620](https://github.com/astronomer/astro-sdk/pull/620)
+* Remove markupsafe from dependencies [#623](https://github.com/astronomer/astro-sdk/pull/623)
+* Added `extend_existing` to Sqla Table object [#626](https://github.com/astronomer/astro-sdk/pull/626)
+* Move config to store DF in XCom to settings file [#537](https://github.com/astronomer/astro-sdk/pull/537)
+* Make the operator names consistent [#634](https://github.com/astronomer/astro-sdk/pull/634)
+* Use `exc_info` for exception logging [#643](https://github.com/astronomer/astro-sdk/pull/643)
+* Update query for getting bigquery table schema  [#661](https://github.com/astronomer/astro-sdk/pull/661)
```

Comment:
```suggestion
```

Reply:
@kaxil this was just for the test case and not our issue with our codebase. I think we can remove this too.

Reply:
Oh yes, you are right